CHANGELOG.next.toml

```diff
@@ -78,3 +78,12 @@ message = "MSRV increased from `1.54` to `1.56.1` per our 2-behind MSRV policy."
 references = ["smithy-rs#1130"]
 meta = { "breaking" = true, "tada" = false, "bug" = false }
 author = "jdisanti"
+
+[[aws-sdk-rust]]
+message = """
+`Client` no longer has generics, and now uses `DynConnector` and `DynMiddleware` to allow for connector/middleware customization.
```

Comment:
can you update the changelog message to clarify that this is all generated service clients? Might also be worth calling out that this simplifies the code and docs. 

The only other call out might be custom retry policies? Not sure.